src/applications/vaos/services/appointment/index.js

```diff
@@ -1069,3 +1082,42 @@ export function getAppointmentTimezone(appointment) {
     description: getTimezoneNameFromAbbr(abbreviation),
   };
 }
+
+/**
+ * Transforms a provider object from the providers endpoint into our
+ * VAOS format
+ *
+ * @export
+ * @param {provider} provider A provider from the providers endpoint
+ * @returns {Provider} A Provider resource
+ */
+export function transformPreferredProviderV2(provider) {
+  return {
+    resourceType: 'Provider',
+    id: provider.providerIdentifier,
+    name: provider.name,
+  };
+}
+
+/**
+ * Fetch provider information
+ *
+ * @export
+ * @param {String} providerNpi An id for the provider to fetch info for
+ * @returns {Provider} Provider info
+ */
+export function getPreferredProviderV2(providerNpi) {
+  return getPreferredCCProvider(providerNpi);
+}
+
+/**
+ * Transform provider information
+ *
+ * @export
+ * @param {String} providerNpi An id for the provider to fetch info for
+ * @returns {Provider} Transformed Provider info
+ */
+export async function getTransformedPreferredProvider(providerNpi) {
+  const prov = await getPreferredProviderV2(providerNpi);
+  return transformPreferredProviderV2(prov);
+}
```

Comment:
I would remove this function and call 'getPreferredProviderV2' directly unless there is a reason for this wrapper function that I missed.